tutorials/models/1_gnn/4_rgcn.py

```diff
@@ -362,4 +362,4 @@ def forward(self, g):
 # The implementation is similar to that presented here, but with an extra DistMult layer
 # stacked on top of the R-GCN layers. You can find the complete
 # implementation of link prediction with R-GCN in our `Github Python code example
-#  <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link_predict.py>`_.
+#  <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link.py>`_.

```

Comment:
as you can see in https://doc.dgl.ai/tutorials/models/1_gnn/4_rgcn.html, the web link is displayed explicitly which is not expected. pls help fix this in this PR. To fix it, just move `example` text in the end of previous line into the same line of `<https:...`.

Reply:
I have made the required changes.